### DISCUSSION TITLE
course和statement绑定了

### DISCUSSION THREAD

**Discussion by PrometheusH** [cuixueshe/earthworm#718] [General]
能不能把course和statement解绑？我有个需求新增加一个类X，这个类X跟statement的地位、作用都是一样的，但course和statement绑定了。

**Comment by PrometheusH** [comment]
警告 警告，误入typescript类型体操，注意安全

**Comment by cuixiaorui** [comment]
没懂 类X 是个啥
加它的意义是什么
